### PR TITLE
Add dependencies to package.json

### LINKED ISSUE #15: Dependencies are not defined in package.json
Currently, this package is published to NPM, but installing it via NPM will not install the appropriate dependencies (i.e. angular, angular-gettext, moment). These dependencies should be defined in package.json.

### PR DESCRIPTION
**Pull request by TheRealJon**
Fixes #15.

## Changes
- Add primary dependencies to package.json
- Add dev dependencies to package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,17 @@
   "version": "0.0.16",
```
Comment (spadgett) [inline package.json]: Do you mind bumping the version, too? Then I'll cut a release when this merges.

Comment (TheRealJon) [reply package.json]: Will do

**Comment by TheRealJon** [comment]
@spadgett version bumped. Should I squash or just leave it so the version bump is in the commit history?

**Comment by spadgett** [comment]
I think a separate commit is fine.

**Review by spadgett — APPROVED** [review]

**Pull request merged**